portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,48 @@
+body{
+    margin:0;
+}
+
+
+#fullpage{
+    min-width: 100vw;
+    min-height: 100vh;
+    min-width: 100vw;
+    min-height: 100vh;
+    overflow-x: hidden;
+    overflow-y: scroll;
+}
+
+.section{
+    min-height: 100vh;
+    max-height: 100vh;
+    overflow: hidden;
+    min-width: 100%;
+    max-width: 100%;
+}
+
+#projects{
+    display: grid;
+    grid-template-rows: repeat(2,1fr);
```

Comment:
use [the `grid-template` shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template) instead (reference: https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)